src/callkit.cs

```diff
@@ -68,6 +68,8 @@ public enum CXErrorCodeCallDirectoryManagerError : nint {
 		MaximumEntriesExceeded = 5,
 		ExtensionDisabled = 6,
 		CurrentlyLoading = 7,
+		[iOS (11, 0)]
```

Comment:
Minor but `[iOS (11,0)]` is prettier (note you sometimes use this in this diff).